nimble/host/mesh/src/prov.c

```diff
@@ -1385,6 +1385,8 @@ static void gen_prov_start(struct prov_rx *rx, struct os_mbuf *buf)
 		gen_prov_ack_send(rx->xact_id);
 		return;
 	}
+	
+	u16_t trailing_space = OS_MBUF_TRAILINGSPACE(link.rx.buf);
```

Comment:
Good catch! Thanks. I have couple of comments:
1) Please prefix commit message with "mesh:". Having this it would bring my attention much faster :)
2) Please explain in commit message why we need this change. Comment from PR will not stay in git.
3) Please move `trailing_space` declaration on top of the function

Reply:
Thanks for your comments, I will add the comments to the commit message.